ui/index.html

```diff
@@ -6,17 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <title>Identity platform</title>
-    <link rel="shortcut icon" href="https://assets.ubuntu.com/v1/49a1a858-favicon-32x32.png" type="image/x-icon" />
+    <link
+      rel="shortcut icon"
+      href="https://assets.ubuntu.com/v1/49a1a858-favicon-32x32.png"
+      type="image/x-icon"
+    />
 
     <script>
-        const global = globalThis;
+      const global = globalThis;
     </script>
+    <% if(process.env.NODE_ENV === "development"){ %>
+    <base href="/ui/" />
+    <% } else{ %>
     <base href="{{ . }}ui/" />
+    <% } %>
```

Comment:
The way we populate this value already takes care of normalizing the path for the slashes, so this is not needed

Reply:
Sorry, which part do you mean isn't needed? This change is for development where the UI is run with Vite and so the `{{ . }}` is not transformed to a value (so the base literally becomes `{{ . }}ui/`).

Reply:
That's ok, then I'll resolve this. Thank you!